CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Feature: Add named export for components. Default export for these components
```

Comment:
Je pense qu'on pourrait lister les composants qui sont affectés par ce changement pour plus de précision. Ou alors dire `Add named export for all components. Default export is now deprecated.`